protocols/token-registry/test/TestTokenRegistry-unit.js

```diff
@@ -0,0 +1,73 @@
+const TokenRegistry = artifacts.require('TokenRegistry')
+const { takeSnapshot, revertToSnapShot } = require('@airswap/test-utils').time
+const { reverted, equal, emitted } = require('@airswap/test-utils').assert
+
+contract('TokenRegistry Unit Tests', async accounts => {
+  const owner = accounts[0]
+  const nonOwner = accounts[1]
+  const erc20asset = accounts[4]
```

Comment:
Why `[4]`?

Reply:
don't have a good reason, making it accounts[2]